src/main/scala/com/databricks/labs/overwatch/ApiMeta.scala

```diff
@@ -80,6 +96,31 @@ trait ApiMeta {
     true
   }
 
+  private[overwatch] def getBaseRequest(): HttpRequest = {
```

Comment:
Nice! :) This is exactly what I was looking for. Good work.

Reply:
thanks:)